ThingiBrowser/ThingiBrowserService.py

```diff
@@ -67,14 +68,22 @@ def __init__(self, extension: "ThingiBrowserExtension", parent=None):
         self._thing_files = []  # type: List[ThingFile]
         self._is_downloading = False  # type: bool
 
+        PreferencesHelper.initSetting(Settings.DEFAULT_API_CLIENT, "thingiverse");
```

Comment:
this line also returns the current or default value of the setting as it is stored in preferences. We can assign that directly to self._active_driver_name (so no extra getSettingValue is needed there).